xcm-support/src/currency_adapter.rs

```diff
@@ -145,6 +145,12 @@ impl<
 		DepositFailureHandler,
 	>
 {
+	fn check_in(_origin: &MultiLocation, _what: &MultiAsset) {}
```

Comment:
Need to align with polkadot CurrencyAdapter: https://github.com/paritytech/polkadot/blob/b0722401e1d66e6b5470995bf748467ea3fb92dd/xcm/xcm-builder/src/currency_adapter.rs#L97-L134

Reply:
why do we need to implement those? I don't think we will ever support teleport operation locally

Reply:
Oh yes, only statemine need this `CurrencyAdapter` implementation, but in our testcase both sender and receiver are using same `MultiCurrencyAdapter`. I'll try let sender use `MultiCurrencyAdapter` and receiver use `CurrencyAdapter`.

Reply:
If `ParaC` using polkadot CurrencyAdapter, there'll be execution error in testcase, So I add another `MultiTeleportCurrencyAdapter` inside mock, this `MultiTeleportCurrencyAdapter` add default `check_in` and `can_check_in` implemenats. It's used only in testcase.